targets/particle/README.md

```diff
@@ -0,0 +1,91 @@
+### About
+
+This folder contains files to run JerryScript beside Particle Device Firmware on Photon board.
+It  runs a mini example, blinking an LED which is the "Hello World" example of the microcontroller universe.
+
+### How to build
+
+#### 1. Preface / Directory structure
+
+Assume `harmony` as the path to the projects to build.
```

Comment:
I think its an accidentally brought folder name from iotjs project. I renamed it to 'root'.